[BUG] sort tslint issue

### Summary:
tslint issue with .sort method.

`error TS2345: Argument of type '"descending"' is not assignable to parameter of type 'SortOrder'.`

### Code sample:
https://github.com/jericopulvera/dynamoose-sort-ts-issue
```
import * as dynamoose from "dynamoose";
import express from "express";

const app = express();
const port = 3000;

dynamoose.aws.sdk.config.update({
  region: "ap-southeast-1",
});

dynamoose.aws.ddb.local();

const Cat = dynamoose.model("Cat", new dynamoose.Schema({ name: String }));

app.get("/", async (req, res) => {
  return Cat.query({ test: 1 }).sort("descending").limit(10).exec();
});

app.listen(port, () => {
  console.log(`listening at http://localhost:${port}`);
});

```


### Current output and behavior (including stack trace):
```
➜ npm run build

> dynamoose-sort-ts-issue@1.0.0 prebuild /Users/eco/Code/dynamoose-ts-test
> tslint -c tslint.json -p tsconfig.json --fix


> dynamoose-sort-ts-issue@1.0.0 build /Users/eco/Code/dynamoose-ts-test
> tsc

index.ts:16:38 - error TS2345: Argument of type '"descending"' is not assignable to parameter of type 'SortOrder'.

16   return Cat.query({ test: 1 }).sort("descending").limit(10).exec();
                                        ~~~~~~~~~~~~


Found 1 error.
```



### Expected output and behavior:
Build successfully


### Environment:

Operating System: macOS
Operating System Version: 11.1
Node.js version (`node -v`): 14.15.4
NPM version: (`npm -v`): 6.14.10
Dynamoose version: 2.7.3


### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [x] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [x] I have tested the code provided and am confident it doesn't work as intended
- [x] I have filled out all fields above
- [x] I am running the latest version of Dynamoose

--- a/lib/DocumentRetriever.ts
+++ b/lib/DocumentRetriever.ts
@@ -34,7 +34,7 @@ abstract class DocumentRetriever {
 		consistent?: boolean;
 		count?: boolean;
 		parallel?: number;
-		sort?: SortOrder;
+		sort?: SortOrder | `${SortOrder}`;
 	};
 	getRequest: (this: DocumentRetriever) => Promise<any>;
 	all: (this: DocumentRetriever, delay?: number, max?: number) => DocumentRetriever;
@@ -328,7 +328,7 @@ export class Query<T> extends DocumentRetriever {
 		return super.exec(callback);
 	}
 
-	sort (order: SortOrder): Query<T> {
+	sort (order: SortOrder | `${SortOrder}`): Query<T> {
 		this.settings.sort = order;
 		return this;
 	}

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "mocha": "^7.2.0",
     "nyc": "^15.1.0",
     "rimraf": "^3.0.2",
-    "typescript": "^4.0.5"
+    "typescript": "^4.3.5"
   },
   "scripts": {
     "prepare": "npm run build:clean && npm run build",

--- a/test/types/model/Query.ts
+++ b/test/types/model/Query.ts
@@ -44,6 +44,8 @@ UserTypedModel.query("name").eq("Will").using("name-index");
 // query.sort(order)
 UserTypedModel.query("name").eq("Will").sort(SortOrder.ascending);
 UserTypedModel.query("name").eq("Will").sort(SortOrder.descending);
+UserTypedModel.query("name").eq("Will").sort("ascending");
+UserTypedModel.query("name").eq("Will").sort("descending");
 
 // query.all([delay[, max]])
 UserTypedModel.query("name").eq("Will").all();

